src/finn/transformation/fpgadataflow/templates.py

```diff
@@ -126,120 +126,113 @@ def __init__(
         # used by TLastMarker to signal end of transmission for AXI CDMA
         self.REG_OFFSET_NUM_ITERS = 0x10
 
-    def load_input(self):
+    def load_input(self, inputfile):
```

Comment:
not sure if this function has much point being included in the driver tbh, it's just np.load
I suggest to add fold_input as a member function and move the input folding into that instead of doing that as part of pack_input